src/views/Predictions.vue

```diff
@@ -82,7 +82,10 @@ export default {
 
   async mounted() {
     if (this.userId) {
-      this.user = await this.fetchUser({ userId: this.userId })
+      this.user = await this.fetchUser({
+        userId: this.userId,
+        ownPredictions: true,
```

Comment:
what exactly is the ownPredictions doing?

Reply:
Just a random property bame, didnt know what to call it. But needed to pass something to tell the fetch to skip the resolve for own User in this case